test-files/cases/attribute/network/highway-2888-reviews-by-score/validation-report

```diff
@@ -0,0 +1,12 @@
+Input: /ramdisk/hoot/test-files/cases/attribute/network/highway-2888-reviews-by-score/Output.osm
```

Comment:
Every case test gets a file, `validation-report`, which acts as the baseline for validation errors in the conflated output.